### PR TITLE
add custom Torus configuration button

### PR DESCRIPTION
**Pull request by dkent600**
Resolves: https://github.com/daostack/alchemy/issues/1540

### CODE REVIEW

**Comment by orenyodfat** [comment]
it say "open metamask configuration" . is that issue #1542 ?

**Comment by dkent600** [comment]
> it say "open metamask configuration" . is that issue #1542 ?

@orenyodfat  Yes

**Comment by orenyodfat** [comment]
should we add tool tip or something near this button..so the user will be aware of that..? it is difficult to find it.

**Comment by dkent600** [comment]
> should we add tool tip or something near this button..so the user will be aware of that..? it is difficult to find it.

The button already has a tooltip.    Where are you looking for the button and not finding it?

**Comment by EzraWeller** [comment]
This looks great to me, was able to import an account and switch between them, with the result showing up in Alchemy.

File: src/layouts/Header.tsx [hunk 1]
```diff
@@ -175,6 +176,7 @@ class Header extends React.Component<IProps, null> {
     const daoAvatarAddress = dao ? dao.address : null;
     const accountIsEnabled = getAccountIsEnabled();
     const web3ProviderInfo = getWeb3ProviderInfo();
+    const web3Provider = getWeb3Provider();
```
Comment (tibetsprague) [inline src/layouts/Header.tsx]: kind of confusing to have both web3ProviderInfo and web3Provider, but ok

**Pull request merged**